todo_app/data/mongodb.py

```diff
@@ -0,0 +1,47 @@
+from todo_app.data.classes import to_do_item
+from pymongo import MongoClient
+from bson import ObjectId 
+import os
+
+key = os.environ.get('CONNECTION_STRING')
+
+client = MongoClient(key) #PRIMARY CONNECTION STRING
+db = client.todo_app    #Select the database
+todos = db.todo_tasks
+
+
+
+def show_cards():
+    tasks=[]
+    for task in todos.find():
+        tasks.append(to_do_item(id=task["_id"], title=task["title"], status=task["status"]))
+    return tasks
+    
+def add_card(name):
+    key = os.environ.get('CONNECTION_STRING')
+    print(key)
```

Comment:
We definitely don't want these lines! You'll be printing your connection string - which includes a DB password - to the console and logging it in Azure; we should get rid of this